WultraMobileTokenSDK/Operations/Model/Requests/WMTAuthorizationData.swift

```diff
@@ -25,8 +25,45 @@ class WMTAuthorizationData: Codable {
     /// Operation id
     let id: String
     
-    init(operationId: String, operationData: String) {
-        self.data = operationData
-        self.id   = operationId
+    /// Proximity Otp Object
```

Comment:
`OTP` should be `uppercased. Also, using "data" instead of "object" would be more fitting.